force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -71,7 +87,15 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Positive", "Negative", or "Zero", or null if the number is null.
      */
     public static String checkNumber(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        String numberVerify;
+        if (a == 0){
+            numberVerify = 'Zero';
+        } else if (a > 0) {
+            numberVerify = 'Positive';
+        } else if (a < 0) {
+            numberVerify = 'Negative';
+        } 
+        return numberVerify; // Replace null with the variable you used to store the result
```

Comment:
Good job. This is an example of where I might use the replace conditional with guard clause pattern I shared in Slack. 

```
 public static String checkNumber(Integer a) {

        if (a == null) {
            return null;
        }
        if (a > 0) {
            return 'Positive';
        }
        if (a < 0) {
            return 'Negative';
        }
            return 'Zero';
    }
 ```